Modules/IO/MINC/src/itkMINCImageIO.cxx

```diff
@@ -997,8 +1007,10 @@ MINCImageIO::WriteImageInformation()
   const vnl_matrix<double> inverseDirectionCosines{ vnl_matrix_inverse<double>(dircosmatrix).as_matrix() };
   origin *= inverseDirectionCosines; // transform to minc convention
 
+
   // Convert ITK direction cosines from PositiveCoordinateOrientation LPS to PositiveCoordinateOrientation RAS
-  dircosmatrix *= RAS_tofrom_LPS;
+  if (this->m_RAStoLPS)
+    dircosmatrix *= RAS_tofrom_LPS;
```

Comment:
Stick to ITK SWG style: medial capitals (`dirCosMatrix`) and do not use underscores.